### DISCUSSION TITLE
Question about reusing of initial stress field by gravity loading

### DISCUSSION THREAD

**Discussion by Tu-Shiqin** [Anura3D/Anura3D_OpenSource#71] [Q&A] [answered]
Hi Dear all,
I want to apply different boundary conditions to the model that has been initialized by quasi-static gravity loading and compare the results. I have tried to copy the .A3D folder and impose different boundary conditions by changing the .GOM and .CPS files. But the program fails to run due to incorrect path to read file. How should I do to reuse the initial model? 

Thank you for any helps.
Tu

**Comment by Tu-Shiqin** [comment]
Please allow me to add one question about the drainage in submerged saturated soil. I try to use the "Phreatic Surface", but it seem doesn't work . The hydrostatic head of bottom  in the model is the  height of saturated soil. How to set the free water surface correctly? Hope for your suggestions.

**Comment by ayerrocolom** [comment, marked as answer]
Dear Tu,

If you want to reuse the files in another folder, you need to edit the calculate.BAT file (right click on it and Edit) and include the path of the new directory and executable.

Submerged analysis will not work together with a phreatic surface, but the whole model is assumed in submerged conditions. In this case, the code uses an equivalent submerged unit weight and only excess pore water pressures are computed.

I hope this helps.
Alba

**Reply by Tu-Shiqin** [reply]
Thank you for your prompt answer. It really helps me. But I haven't figured out the problem about drainage in submerged soil, perhaps because I haven't explained the problem clearly. If the option of "Submerged Calculation" is not open and the phreatic surface located above the saturated soil surface is applied, Is the hydrostatic head equal to the height of the phreatic surface or soil surface? (The bottom of soil is selected as the datum of the position head)

**Reply by Tu-Shiqin** [reply]
I have tried to simulate a saturated soil model with phreatic surface. The results show that the hydrostatic head is equal to the height of soil surface.

**Reply by Tu-Shiqin** [reply]
And I found that there is an option "Reset displacement" in the "Calculation Data"dialogue box. Will the displacement of the material points be reset after each calculation step (or load step)?
